bin/pull_condor_rpms.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# only need to fetch a subset of packages from condor repos
+CONDOR_PKGS=(condor htcondor-ce minicondor python3-condor)
```

Comment:
I think we want everything, or at least for OSG 23 we do.
We will definitely need the OSDF-Client components, which are "stashcp", "condor-stash-plugin" (for now), and "condor-osdf-plugin" (once we start making that). We do not have the source RPMs that those were created from, so you can skip those.